examples/server/tests/features/steps/steps.py

```diff
@@ -285,17 +285,17 @@ def step_seed(context, seed):
 
 @step(u'a prefix prompt')
```

Comment:
u'' and u"" strings are a Python 2 construct. In Python 3, string literals are always unicode.

Reply:
thanks, brhave generates this automatically, will remove